lib/components/common/StyledIconButton.tsx

```diff
@@ -1,13 +1,13 @@
-import { styled } from '@mui/material/styles';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them